proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+
+## 6. General Assembly
+
+1. The General Assembly consists of all active members, each with one vote.
+2. It takes place at least once a year, with invitations sent at least six weeks prior.
+3. An extraordinary General Assembly may be called if requested by at least 20% of members.
+4. The General Assembly has a quorum if the invitation was submitted in proper form.
+5. A ¾ majority of votes is required to amend the Articles of Association or decide the liquidation of the Association.
+6. Active members unable to attend may be represented by another member with written authorization.
+
+## 7. Duties of the General Assembly
+
+1. The General Assembly is the highest decision-making organ and is responsible for all duties unless otherwise assigned.
+2. It elects the Board and can deselect board members.
+3. It decides on member expulsion objections.
+4. It receives the Board’s annual report and auditor’s report, and discharges the Board’s responsibilities.
+5. It decides on amendments to the Articles of Association and liquidation of the Association.
+6. It presents annual accounts and reports, and appoints auditors.
+7. It decides on purchases, sales, investments, and raising loans.
+
+## 8. The Board
+
+nsists of the Chairman, the Second Chairman, and the Treasurer. 
+
+1. The Board consists of the Chairman, the Second Chairman, and the Treasurer elected for a 3-year term.
+2. The Board’s duties and meeting procedures are stipulated in the by-laws.
+3. The Board decides on all matters unless a General Assembly decision is required.
+4. Each Board member has the right to represent the Association.
+5. Provisional board members are appointed in case of a vacancy until the next General Assembly.
+6. The Board can appoint a full-time director to manage current affairs.
+7. The director participates in General Assemblies and Board meetings.
+8. The Board can amend the Articles of Association for formal reasons as required by authorities.
+
+## 9. By-Laws
+
+The by-laws govern all details of the Association’s activities not covered in the Articles of Association. The Board implements the by-laws following member approval.
+
+## 10. Meeting Minutes
+
+All decisions by the Board and General Assembly are documented in writing and available for members’ perusal.
+
+## 11. Labour Agreements
+
+The Board regulates remuneration for full-time employees in the by-laws.
+
+## 13. Financing of the Association
+
+1. The Association is funded through membership fees, public contributions, donations, and other financial contributions.
+2. Membership fees are decided by the General Assembly or through an internet vote.
+3. In case of dissolution or liquidation, all assets will go to a suitable non-profit organization for non-profit purposes.
+
+## 14. Inception of the Articles of Association
+
+These Articles of Association came into effect following a General Assembly decision and replace any previous Articles effective since their date of establishment.
+
```

Comment:
These are all honorable goals, but I don't think they belong here as we can't guarantee to uphold all of them, nor even verify.

Reply:
>  Not the name nor any part of Mixxx can be sold or commercialised without a written official permission of all previous contributors (broad sense) .
>  A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members.
>  Mixxx may never connect it's name to any comercial organisation, it's products may never be used for commercial activities.

This would require registering a trademark iirc. Certainly possible, just nothing we have done now. See [the article we published](https://mixxx.org/news/2020-05-22-you-dont-need-to-pay-for-mixxx/) for more background.

>  Mixxx products may never be abused to violate peoples privacy in any possible way.

Very broad, almost unenforceable IMO.

>  Mixxx will avoid getting involved in political movements,

define political movements? also we do usually build out a census in the team [and](https://mixxx.org/news/2022-03-29-russians-war/) [take](https://mixxx.org/news/2021-03-26-fsf-statement/) [a stance](https://mixxx.org/news/2020-06-29-black-lives-matter/).

>  Mixxx will honour human and animal rights, zero waste policy and will attempt to be carbon neutral.

Ofcourse, but how far does that extend? Since we're using github and github is owned by microsoft, are we affiliated with all the awful shit microsoft has done? Even [github themselves have blood on their hands](https://www.theverge.com/2019/10/9/20906213/github-ice-microsoft-software-email-contract-immigration-nonprofit-donation)

Reply:
* It's always better to start with written goals in a startup-declaration, even if they are sky high. It's sets the code of conduct to strive for. Changing or adapting something like this later will always involve a discussion.
* Political not in the sense of what is common (war in Ukraine/Gaza; Human rights in China/Iran, capital punishment in US, Japan... ,lgbtqia+ rights) but as in supporting a political party or candidate.
* What can be obvious today can be very strange in some years. Mentioning obvious things as a code of conduct to strive for sets the ideas for successors.
* I know the article and I have read it multiple times, it's part of the reason I'm here and writing this.
* Maybe investigating a trademark deposal could be a task for the new board?
* Violating privacy = strive: not cooperating on privacy violating tools. (brand new future :-) ), like all kind of aps that collect all kind of information to feed the big data monster.
* As we eat and drink we're already part of a lot of shit. It's a strive.

*** The only guarantee is the commitment. Maybe another task for the future board is explaining our conduct in the annual report as a kind of mirror. 
*** I repeat: it's 100 times easier to start with a written code of conduct than ever add or adjust this.
*** Writing this down is also a message to all contributors and users, a kind of fairtrade label.

Reply:
> Not the name nor any part of Mixxx can be sold or commercialised without a written official permission of all previous contributors (broad sense) .
A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members.

This would also mean that we cannot sell Mixxx t-shirts to fund development/hosting without having written permission ftom every past contributor. Since Mixxx is already 20 years old, some might even have passed away by now. I think this is way too strict.

I suppose you want to prevent transferring mixxx to a for-profit company? I agree that this would be nice, but the code is already protected by GPL, so the community may fork. And the trademark would be nice to protect, but I'm unsure if we can enforce that in the statue of the association.

Reply:
Thank you for this good discussion. I think I understand the base idea of Eve's proposal to state obvious things in the "Articles of association" because in the event that once is bending it we can use the written statements to prevent it. 

The difficult part is, to do it in the right extend. 

The conflict is here that we not have yet the e.V. to transfer all legal stuff from individuals to the e.V. 
It will be an exiting time to go to the process. 

I will try to incorporate the idea in a following commit. Please verify.

Reply:
I have added a sentence about building an inclusive community and about poitical neutrality. 

> the spirit, philosophy and independence of the entity and it's products.

Since the Mixxx e.V. covers mainly the non-profit education purpose, this does not fit yet. 
Once we have transferred the trademark we may add some sentences about that, on the other hand later changes of the document are expensive because of the legal obligations. We may add such to the by-law? 

> A commercial engagement with a third party can only be granted in order to supply merchandise towards the members, or for occasional activities for members. Mixxx may never connect it's name to any comercial organisation, it's products may never be used for commercial activities.

I would love to have a commercial part that is able to employ a full time worker. These activities shall not against the Terms of the GPL and the terms of this document. So I think this is kind of covered? 

> Mixxx products may never be abused to violate peoples privacy in any possible way.

This should be covered sufficient by 14. and 2.2.6 without mentoring Mixxx itself. 

> Mixxx will honour human and animal rights, zero waste policy and will attempt to be carbon neutral. 

This is important, but hard to archive using GitHub build servers is the opposite of carbon neutral. 
So I would not make it part of our Satzung.

Reply:
I like the inclusive sentence.
Watch out with 'should be covered', depending on laws, general terms and articles that can be changed by third parties. Concrete philophy, ideas on paper can't.

Reply:
You mean the data protection, right? 
How could we make that more explicit. 

Mixxx e.V. will not publish software that does not follw the EU data protection rules ..

Reply:
Even the EU GDPR is not a bible, a change of coalition in the EP, a new majority and articles can be changed. The discussion about AI in EP is going that way. The first drafts were much better, half of the original ruleshad to leave because of economical aspects.'Yes but it should be possible to ....'
In the end profit came before 'what is right'.
Now's the discussion about anonimity on the internet.
Look what happens in the US, there are Big Tech companies funding a certain candidate because he'would change some laws, another wealthy man leaves California because of the 'inclusive laws'.
Counting on general laws and regulations isn't as good as stipulating what you want yourself.

We could write something as: 
We honour the choice of our contributors and users to participate or use Mixxx in complete anonimity  or with complete identifications on Mixxx websites, discussion fora and development platform.  Who,, how, wherefore  and with whom people use our software is not in our interest. Mixxx honours letter secrecy for all communication between developers and users. 
Mixxx will never gather information about it's users without their knwoledge, fi to offer support.

Reply:
I have tried to incoperate your suggestions. Please have a look.